frontend/plugins/insurance_ui/src/modules/insurance/components/insuranceTypes/InsuranceTypesColumns.tsx

```
@@ -14,7 +15,10 @@ import {
 const attributesColumn: ColumnDef<InsuranceType> = {
   id: 'attributes',
   accessorKey: 'attributes',
-  header: () => <RecordTable.InlineHead icon={IconList} label="Attributes" />,
+  header: () => {
+    const { t } = useTranslation('insurance');
```

ESLint

[error] React Hook "useTranslation" is called in function "header" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)